### ISSUE TITLE
./Scripts/bootstrap.sh error

### ISSUE THREAD

**Issue by wjl025** [facebook/WebDriverAgent#394]
why I am git clone the last version WebDriverAgent,but run ./Scripts/bootstrap.sh

ime: 114ms
   [0] multi main 28 bytes {0} [built] [1 error]
    + 1 hidden modules

ERROR in ./js/app.js
Module parse failed: /Users/jialinwang/git/appium/node_modules/appium-xcuitest-driver/WebDriverAgent/Inspector/js/app.js Unexpected token (59:4)
You may need an appropriate loader to handle this file type.
SyntaxError: Unexpected token (59:4)
    at Parser.pp$4.raise (/Users/jialinwang/git/ap

**Comment by marekcirkos** [comment]
@wjl025 Can you checkout WDA separately and check if it is failing for you?

**Comment by wjl025** [comment]
@marekcirkos 
success:
/Users/jialinwang/git/WebDriverAgent
fail:
/Users/jialinwang/git/appium/node_modules/appium-xcuitest-driver/WebDriverAgent

all checkout WDA separately, only path  differ

**Comment by marekcirkos** [comment]
@wjl025 You should poke Appium guys it is clear that there is some difference here.

**Comment by wjl025** [comment]
@marekcirkos 
ok,thanks

**Comment by toejough** [comment]
some additional info:

I recently ran into this again, and did an experiment - I copied the Appium WDA up to my home directory and ran the boostrap, and everything worked.

This tells me there's no actual version difference - the failure must somehow stem from WDA being nested inside of appium's directory structure.  That indicates that the bootstrap script is sensitive to files/dirs in the enclosing directories.  That seems generally bad, and it feels like this should be fixed on the WDA side, but I say that without understanding how the bootstrapping or the tools it uses works, so maybe it's still something better fixed in appium?  `¯\_(ツ)_/¯`

@wjl025 - did you open an issue with Appium?  ^ might be relevant to the resolution.
